<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AgProperties</class>
 <widget class="QDialog" name="AgProperties">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>434</width>
    <height>342</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>290</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBoxMutacja">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>20</y>
     <width>121</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>Mutacje</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QRadioButton" name="radioButtonM1">
      <property name="text">
       <string>1 punktowa</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButtonM2">
      <property name="text">
       <string>2 punktowa</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButtonML">
      <property name="text">
       <string>losowa</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBoxKrzyzowanie">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>121</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>Krzyżowanie</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QRadioButton" name="radioButtonK1">
      <property name="text">
       <string>1 punktowe</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButtonK2">
      <property name="text">
       <string>2 punktowe</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButtonKW">
      <property name="text">
       <string>wielo pónkowe</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBoxSelekcja">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>20</y>
     <width>115</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>Selekcja</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QRadioButton" name="radioElitarna">
      <property name="text">
       <string>Elitarna</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioTurniejowa">
      <property name="text">
       <string>Turniejowa</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioLosowa">
      <property name="text">
       <string>Losowa</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioProporcjonalna">
      <property name="text">
       <string>Proporcjonalna</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>243</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="labelLimitEpok">
        <property name="text">
         <string>Limit epok</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditLimitEpok">
        <property name="text">
         <string>500</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="proportionLabel">
        <property name="text">
         <string>ilosc do selekcji</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditProportion">
        <property name="text">
         <string>5</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AgProperties</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AgProperties</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
